script.js

```diff
@@ -10,6 +10,9 @@ quoteBtn.addEventListener("click", generateQuote);
 generateQuote();
 
 function generateQuote() {
+  var quoteText = "";
+  var quoteAuthor = "";
+
```

Comment:
Moved the `quoteText` and `quoteAuthor` variables out of the method call (`then(....)`) and out in the main function, so that it is scoped properly for `shareTweet() ` to be able to read them.